### ISSUE TITLE
twisted.mail.test.test_imap.IMAP4ServerTests.testPartialAppend fails if the install isn't writable.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8277]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)| @tomprince reported|
|-|-|
|Trac ID|trac#8277|
|Type|enhancement|
|Created|2016-04-07 17:06:45Z|
|Branch|https://github.com/twisted/twisted/tree/notouch-8277|

It touches the message for no apparent reason.

Attachments:

* [partial-append.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/397/3974f142b060227a9124cfc95de4288be821d728/30204505963e2376e5ffe0cae9465df4ec318954.patch) (582 bytes) - added by tom.prince on 2016-04-07 17:07:00Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__8277 8277
type__enhancement enhancement
reporter__tom_prince tom.prince
priority__normal normal
milestone__None None
branch__branches_notouch_8277 branches/notouch-8277
branch_author__glyph glyph
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1460048805250185 1460048805250185
changetime__1460761877349855 1460761877349855
version__None None
owner__glyph glyph

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_2"></a>|@glyph commented|
|-|-|

(In [47254]) Branching to notouch-8277.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_3"></a>|@glyph set owner to @glyph|
|-|-|

Thanks.  Looks fine, modulo the one pyflakes error.  I'll land.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph set status to `closed`|
|-|-|

(In [47258]) Merge notouch-8277: don't touch installed files

Author: tom.prince

Reviewer: glyph

Fixes: [#8277](https://github.com/twisted/twisted/issues/8277)

twisted.mail.test.test_imap.IMAP4ServerTests.testPartialAppend was
touching an installed file for no discernable reason; remove that
unnecessary code.
